src/videojs-http-streaming.js

```diff
@@ -617,6 +618,20 @@ class VhsHandler extends Component {
     });
 
     this.on(this.tech_, 'play', this.play);
+
+    if (this.options_.useNetworkInformation) {
+      this.networkInformation = window.navigator.connection || window.navigator.mozConnection || window.navigator.webkitConnection;
+
+      if (this.networkInformation) {
+        this.networkInformation.addEventListener('change', () => this.setNetworkInformationStats_());
+
+        this.setNetworkInformationStats_();
+      }
+    }
+  }
+
+  setNetworkInformationStats_() {
+    this.effectiveBandwidthInBits_ = this.networkInformation.downlink * 1000 * 1000;
```

Comment:
Suggest add a comment linking the API doc and original unit megabit/s

Reply:
Added